pkg/commands/workload_get_test.go

```diff
@@ -359,6 +367,99 @@ Source
 type:    image
 image:   docker.io/library/nginx:latest
 
+Supply Chain resources not found
+
+No pods found for workload.
+`,
+		},
+		{
+			Name: "show resources",
+			Args: []string{workloadName},
+			GivenObjects: []client.Object{
+				parent.
+					SpecDie(func(d *diecartov1alpha1.WorkloadSpecDie) {
+						d.Source(&cartov1alpha1.Source{
+							Git: &cartov1alpha1.GitSource{
+								URL: url,
+								Ref: cartov1alpha1.GitRef{
+									Branch: "master",
+									Tag:    "v1.0.0",
+									Commit: "abcdef",
+								},
+							},
+						})
+					}).
+					StatusDie(func(d *diecartov1alpha1.WorkloadStatusDie) {
+						d.ConditionsDie(
+							diecartov1alpha1.WorkloadConditionReadyBlank.
+								Status(metav1.ConditionFalse).Reason("OopsieDoodle").
+								Message("a hopefully informative message about what went wrong").
+								LastTransitionTime(metav1.Time{
+									Time: time.Date(2019, 6, 29, 01, 44, 05, 0, time.UTC),
+								}),
+						)
+						d.Resources([]cartov1alpha1.RealizedResource{
+							{
+								Name: "source-provider",
+								Conditions: []metav1.Condition{
+									{
+										Type:   "Ready",
+										Status: "True",
+									},
+									{
+										Type:   "ResourceSubmitted",
+										Status: "True",
+									},
+								},
+							}, {
+								Name: "deliverable",
+								Conditions: []metav1.Condition{
+									{
+										Type:   "Ready",
+										Status: "Unknown",
+									},
+									{
+										Type:   "ResourceSubmitted",
+										Status: "Unknown",
+									},
+								},
+							}, {
+								Name: "image-builder",
+								Conditions: []metav1.Condition{
+									{
+										Type:   "Ready",
+										Status: "False",
+									},
+									{
+										Type:   "ResourceSubmitted",
+										Status: "False",
+									},
+								},
+							},
+						}...)
+					}),
+			},
+			ExpectOutput: `
+# my-workload: OopsieDoodle
+---
+lastTransitionTime: "2019-06-29T01:44:05Z"
+message: a hopefully informative message about what went wrong
+reason: OopsieDoodle
+status: "False"
+type: Ready
+
+Source
+type:     git
+url:      https://example.com
+branch:   master
+tag:      v1.0.0
+commit:   abcdef
+
+RESOURCE          READY     TIME
+source-provider   True      <unknown>
```

Comment:
Add test to assert on `TIME`
```
source-provider   True      <unknown>
```

Reply:
TIME is calculated with the `time.Now()` function, all the tests I've seen so far are with `<unknown>` when using the `TimeStampSince()` func